ios/RNCCameraRollManager.m

```diff
@@ -287,8 +287,12 @@ static void RCTResolvePromise(RCTPromiseResolveBlock resolve,
              @"group_name": [assetCollection localizedTitle],
              @"image": @{
                  @"uri": uri,
+                 @"fileName": [asset valueForKey:@"filename"],
+                 @"originalRotation": [asset valueForKey:@"exifOrientation"],
```

Comment:
originalRotation, type and fileSize are missing from the flow type https://github.com/react-native-community/react-native-cameraroll/pull/30 and the typescript interfaces: https://github.com/react-native-community/react-native-cameraroll/pull/30